alws/crud/build.py

```diff
@@ -88,6 +95,36 @@ async def create_build(
     await db.commit()
     await db.refresh(db_build)
     start_build.send(db_build.id, build.model_dump())
+    if settings.github_integration_enabled:
```

Comment:
I'm wondering if we should do the same as with errata, this is, not updating tasks status until the background task is done. In this case, after the build has 'really' started at the very end of the dramatic task